docs/tutorials/services/data-mlmodel-tutorial.md

```diff
@@ -0,0 +1,322 @@
+   If you do not see a functioning camera stream, verify that you have configured your [camera](/components/camera/) correctly.
+
+1. Then, select the [**DATA** page](https://app.viam.com/data/view) from the top of the screen.
+
+   {{< imgproc src="/tutorials/data-management/data-captured-images.png" alt="The data page showing a variety of captured images with search options presented on the left" resize="1200x" >}}
+
+   Here you can see the images captured so far from the camera on your smart machine.
+   New images should appear roughly every six seconds as cloud sync uploads them from your smart machine.
+
+You can use the filters on the left side of the screen to filter your images by robot, component, date range, and more.
+You can also use the **Copy export command** button to use the Viam CLI to [export captured data](/manage/cli/#data), if desired.
+
+For more information see [View and filter data](/manage/data/view/).
+
+## The ML model service
+
+Once your smart machine is capturing and syncing images to the Viam app, you are ready to train a new machine learning (ML) model using those image.
+You can use an ML model to help your smart recognize and adapt its behavior to the world around it.
+
+For this tutorial, you will train an ML model to be able to recognize specific objects, and then deploy that model to your smart machine using the _ML (machine learning) model service_.
+With a model deployed to your smart machine, the [ML model](/services/ml/) service can be used together with the [vision](/services/vision/) service to analyze newly-detected objects for a possible match to a known object.
+
+To train a new model from your captured data, first tag your images with appropriate labels, train a new model based on those labels, then deploy the model to your smart machine.
+
+### Tag images
+
+1. From the [**DATA** page](https://app.viam.com/data/view) in the Viam app, select an image captured from your smart machine that you would like to tag.
+1. In the **Tags** field on the right-hand side, enter a new tag describing the object in the image that you want your smart machine to be able to identify, then click **Add as new tag**.
+   Tag names support alphanumeric characters, underscores, and hyphens.
+
+   For this tutorial, you will use the objects shown in the [Try Viam](https://app.viam.com/try) test octagon, which include shapes like a red star and an orange triangle.
+   The image below shows adding the `red_star` tag to an image:
+
+   {{< imgproc src="/tutorials/data-management/add-tag-red-star.png" alt="The tags and labels configuration pane of a selected image, with the text red_star entered as the tag" resize="350x" >}}
+
+1. Repeat this process for other images that contain objects you want your model to be able to identify.
+   Once you have added a tag to an image, you can select that tag from the **Tags** drop down menu for other images that also feature the tagged object.
```

Comment:
```suggestion
   Once you have added a tag to an image, you can select that tag from the **Tags** dropdown menu for other images that also feature the tagged object.
```
I think this makes it a compound noun, or "drop-down"..?

Reply:
we currently use drop down menu in other parts of the docs, but we also try to follow the rackspace style guide which does use dropdown. So let's go with that